src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -186,19 +191,34 @@
             join))
         joins))
 
+(defn- matching-field-in-source-query* [source-query field-clause & {:keys [normalize-fn]
+                                                                     :or   {normalize-fn normalize-clause}}]
+  (let [normalized (normalize-fn field-clause)
+        exports    (filter (partial mbql.u/is-clause? :field)
+                           (exports source-query))]
+    ;; first look for an EXACT match in the `exports`
+    (or (some (fn [a-clause]
+                (when (= (normalize-fn a-clause) normalized)
+                  a-clause))
+              exports)
+        ;; if there is no EXACT match, attempt a 'fuzzy' match by disregarding the `:temporal-unit` and `:binning`
```

Comment:
Does it make sense to log something at trace level for possible QP debugging purposes?

Reply:
I did have tracing in here at one point but it was pretty noisy because there are a lot of Fields that go thru this code so I took it out because it was distracting from other stuff I was trying to look at.

I think maybe we can add it back in somewhere better if we need it in the future?